packages/test-react15/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "test-react15",
+  "version": "1.0.0",
+  "description": "React 15 dependencies for testing",
+  "private": true,
+  "dependencies": {
+    "enzyme-adapter-react-15": "^1.0.5",
+    "react": "^15.6.0",
```

Comment:
since react 16 is already released, should we just pin this to the final 15.x version?